.github/workflows/generate-images.yml

```diff
@@ -33,6 +33,15 @@ jobs:
             target: crda-python
             tag: 3.8
             path: Dockerfiles/Dockerfile.python
+          - ecosystem: golang
```

Comment:
According to CRDA standards we need to provide images for major version of each ecosystems. like in Python we have separated image tagged for version 3.6, 3.7, 3.8.
If you are not planning to provide images for all versions then at least tag the current version of ecosystem later we can create image for other version.
So users can select image for version they want to use.